### PR TITLE
fixes "what are you looking for" box 

### PR DESCRIPTION
**Pull request by Danwhy**
ref #146
old:
screen shot 2017-05-11 at 11 29 44
new: 
screen shot 2017-05-11 at 15 33 49

### CODE REVIEW

**Review by samhstn — APPROVED** [review]
Fab

**Review by nelsonic — APPROVED** [review]
@Danwhy LGTM! 👍

File: web/templates/homepage/index.html.eex [hunk 1]
```diff
@@ -59,18 +59,21 @@
   <% end %>
 </section>
 
-<section class="pa4 tc-ns pa1 ph3 pb4 pb5-l" id="suggestion_form">
-  <div class="br1 shadow-2 ph3 pt3 pb2 mt4 tl w-60-ns center">
+<section class="pa4 tc-ns tl-m pa1 ph3 ph5-m ph6-l pb4 pb5-l w-100" id="suggestion_form">
```
Comment (SimonLab) [inline web/templates/homepage/index.html.eex]: do we want the width to be at 100% on desktop?
screenshot from 2017-05-11 16-02-59

**Comment by SimonLab** [comment]
@Danwhy looks good, just one questions about the width, I don't know if we want to apply a width limit on desktop (it's still look good with 100% width)

File: web/templates/homepage/index.html.eex (hunk 1 above)
Comment (Danwhy) [inline web/templates/homepage/index.html.eex]: The text needs to align with the text at the very top, and the box needs to align with that

**Review by SimonLab — APPROVED** [review]
:+1:

**Pull request merged**